Its.Log/ObservableExtensions.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Its.Log.Instrumentation
 {
-    internal static class ObservableExtensions
+    public static class ObservableExtensions
```

Comment:
Why make this public?

Reply:
If I keep this as internal, in ObservableTests.cs line 121 I will see error because the methods defined in the internal ObservableExtensions are not accessible from another assembly (Its.Log.UnitTests). 
![image](https://user-images.githubusercontent.com/6925089/58490396-f2dd3d00-8121-11e9-993b-3c2bcccf32d2.png)

It used to work before but I don't understand why.

Reply:
It's better to keep this internal. One way to test this class is to link it into the test project, which I generally prefer over `InternalsVisibleTo` as it's a more scoped approach that prevents overtesting of internals.

Reply:
I linked ObservableExtensions.cs in UnitTests and made this internal again.

I can't do this for LogEntry.cs because I will define LogEntry twice (once in my linked LogEntry.cs, once in my imported Its.Log project).